.github/workflows/build.yml

```diff
@@ -47,6 +41,17 @@ jobs:
         ./waf.bat configure -T debug
         ./waf.bat build
 
+  build-windows-amd64:
+    runs-on: windows-2019
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build windows-i386
```

Comment:
Be careful! The name is wrong, it isn't i386, it's amd64.

Reply:
shit happens